Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs

```diff
@@ -1438,8 +1474,20 @@ eval (DriverInfoCardActionController (DriverInfoCardController.PrimaryButtonAC P
     ]
 eval (DriverArrivedAction driverArrivalTime) state =
   if any (_ == state.props.currentStage) [ RideAccepted, ChatWithDriver] then do
-      _ <- pure $ setValueToLocalStore DRIVER_ARRIVAL_ACTION "TRIGGER_WAITING_ACTION"
-      exit $ RefreshHomeScreen state { data { driverInfoCardState { driverArrived = true, driverArrivalTime = getExpiryTime driverArrivalTime true } } }
+      if state.data.vehicleVariant /= "DELIVERY_BIKE" then do
+        _ <- pure $ setValueToLocalStore DRIVER_ARRIVAL_ACTION "TRIGGER_WAITING_ACTION"
+        exit $ RefreshHomeScreen state { data { driverInfoCardState { driverArrived = true, driverArrivalTime = getExpiryTime driverArrivalTime true } } }
+      else
+        continue state { data { driverInfoCardState { driverArrived = true } } }
+    else continue state
+
+eval (DriverReachedDestinationAction driverReachedDestinationTime) state =
+  if any (_ == state.props.currentStage) [ RideStarted] then do
+      if state.data.vehicleVariant /= "DELIVERY_BIKE" then do
```

Comment:
this is added for DELIVERY_BIKE case or for other categories
? 
Please verify `/=` once here.

Reply:
for other categories